1.17/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM golang:1.17.9-alpine3.15
+FROM golang:1.17.10-alpine3.15
```

Comment:
In docker hub there's an image with an updated base OS too (`1.17.10-alpine3.16`). Is it possible for renovate bot to also propose upgrade to that and not just `1.17.10`?